rllib/env/wrappers/multi_agent_env_compatibility.py

```diff
@@ -0,0 +1,76 @@
+from typing import Optional, Tuple
```

Comment:
New wrapper class to be used for RLlib's own old MultiAgentEnv classes (abiding to old gym API) to become compatible with the new API.

Analogous to `gymnasium.wrappers.EnvCompatibiltiy`.